toolz/curried/operator.py

```diff
@@ -0,0 +1,28 @@
+import operator
```

Comment:
I need to add `from __future__ import absolute_import` to make this work in Python 2.  Otherwise, there is weird behavior going on and it imports itself (which doesn't have anything defined).